Formula/nuclei.rb

```diff
@@ -0,0 +1,38 @@
+class Nuclei < Formula
+  desc "HTTP/DNS scanner configurable via YAML templates"
+  homepage "https://nuclei.projectdiscovery.io/"
+  url "https://github.com/projectdiscovery/nuclei/archive/v2.2.0.tar.gz"
+  sha256 "c503b565230f5375339c72872d496137abf8be26c156e20447d1ae0752169e40"
+  license "MIT"
+  head "https://github.com/projectdiscovery/nuclei.git"
+
+  depends_on "go" => :build
+
+  def install
+    cd "v2/cmd/nuclei" do
+      system "go", "build", *std_go_args, "main.go"
+    end
+  end
+
+  test do
+    (testpath/"test.yaml").write <<~EOS
+    id: homebrew-test
+    info:
+    name: Homebrew test
+    author: bleepnetworks
+    severity: INFO
+    description: Check DNS functionality
+    dns:
+      - name: "{{FQDN}}"
+      type: A
+      class: inet
+    recursion: true
+    retries: 3
+    matchers:
+      - type: word
+        words:
+          - "IN\tA"
+    EOS
```

Comment:
Sorry, please increase the indentation of everything in the `EOS` block by one level, thanks.

Reply:
Actually, it's fine, I can fix that myself.